src/sgp/Cursor_Control.cc

```diff
@@ -357,3 +362,16 @@ void SetExternMouseCursor(SGPVObject const& vo, UINT16 const region_idx)
 	guiExternVo         = &vo;
 	gusExternVoSubIndex = region_idx;
 }
+
+// Checks if uiCursorIndex is VIDEO_DEFAULT_TO_NO_CURSOR. If so, we parse it as VIDEO_NO_CURSOR or 0 (generic cursor)
+UINT32 ReturnCursorIndex(UINT32 uiCursorIndex)
+{
+	if (uiCursorIndex == VIDEO_DEFAULT_TO_NO_CURSOR)
+	{
+		if (gamepolicy(always_show_cursor_in_tactical))
+			return 0;
```

Comment:
This is NO_UICURSOR, right? Would be cleaner with a named constant.

Reply:
It's a variant of VIDEO_NO_CURSOR (-1). I wasn't sure what to call it. Depending on the new config var, it's parsed as a generic cursor (0) or VIDEO_NO_CURSOR.

Reply:
But that's what I'm saying, isn't the generic cursor NO_UICURSOR?

Reply:
Oh, sorry, I misunderstood your first question.

I wrote 0 as that's what I noticed the list in gUICursors[] used for generic cursor. Looking again, that list randomly uses either CURSOR_NORMAL or 0.

I'll change my 0 to CURSOR_NORMAL.